src/containers/Dashboard.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from "react-redux";
+import Dashboard from "../components/Dashboard";
+
+const mapStateToProps = state => {
+	return {
+		cars: state.cars,
+		user: state.user,
+		email: state.email
```

Comment:
You don't need `email` here.  There's no *toplevel* `email` entry in the store (see `src/redux/state.js` there is only `cars` and `user`).  You're also not using `props.email` in the `Dashboard` component.  I'm guessing this crept in when you were experimenting w/ getting things to work but I wanted to point it out just in case you were confused.